WordScramble.js

```diff
@@ -1,14 +1,15 @@
 const words = ["apple", "plane", "orange", "tiger", "soccer", "basketball", "whale", "water", "lebron", "lion", "hippo", "bottle", "kyrie"];
-
+let random;
 function randomWord(){
```

Comment:
This function name doesn't tell me anything. What this function is doing is setting or choosing a word to be guessed. So something like "setWordToBeGuessed()" would be more informative.